### ISSUE TITLE
RC. Manual Keyboarding Validation not Covered by E2E.

### ISSUE THREAD

**Issue by giancorderoortiz** [SAP/spartacus#10081]
Given new functionality for 3.0, mostly in B2B, 

1. Detail what keyboarding testing is required given that it's not yet covered by e2e tests. 
2. Is mobile testing required? 
3. If manual testing required, please propose how to split up task  (browsers, functionality, device, etc... )

**Comment by developpeurweb** [comment]
1. Just usual tests that are done as a mouse user, but should be possible using the keyboard-only.
2. No mobile tests required for 3.0.
3. Same as the first answer.

All functionality possible for mouse users should be available to keyboard-only users (tabbing to and from buttons and links and activating them for the purpose of checking out orders, cancelling, approving and rejecting. As well as adding, disabling and editing business units, users, cost centers, budgets, limits and groups.

**Note:** On a quick tour of all B2B new areas, I can validate that all interactions are possible with the keyboard and no keyboard traps were detected. **This is not a blocker for 3.0 release.**

 That said, specifically for _MY COMPANY_ sections using sliding panels: the keyboard flow (tabbing order, sequence) and focus management must be improved, currently the user needs to tab A LOT to move around.

- Separate ticket for this issue: #10172
